testing/burp/APKBUILD

```diff
@@ -0,0 +1,64 @@
+# Maintainer: Bernhard J. M. Gruen <bernhard.gruen@googlemail.com>
+pkgname=burp
+pkgver=2.1.28
+pkgrel=0
+pkgdesc="Burp is a network backup and restore program"
+url="http://burp.grke.org"
+arch="all"
+license="AGPL3"
+depends=""
+depends_dev=""
+makedepends="uthash-dev libressl-dev zlib-dev librsync-dev acl-dev"
+install=""
+subpackages="$pkgname-doc $pkgname-server"
+source="https://sourceforge.net/projects/$pkgname/files/$pkgname-$pkgver/$pkgname-$pkgver.tar.bz2 burp.init"
+builddir="$srcdir/$pkgname-$pkgver"
+options="!check"
+
+build() {
+	cd "$builddir"
+	./configure \
+		--prefix=/usr \
+		--sbindir=/usr/bin \
+		--sysconfdir=/etc/burp \
+		--mandir=/usr/share/man \
+		--localstatedir=/var
+	make
+}
+
+package() {
+	cd "$builddir"
+	make DESTDIR="$pkgdir" install-all
+	rm -rf "$pkgdir"/var
+	rm -rf "$pkgdir"/etc/burp/autoupgrade
+	rm -rf "$pkgdir"/etc/burp/clientconfdir
+	chmod -R go-rwx "$pkgdir"/etc/burp
+}
+
+server() {
+	pkgdesc="burp is a network backup and restore program - server configuration and helper scripts"
+	depends="bash"
```

Comment:
Looks like you added bash for burp_ca which is a rather simple shell script. Would be nice if you could check/remove bash-ism and upstream a patch.

Reply:
There are more scripts in /usr/share/burp/scripts that use bash. I will contact the author.